contracts/IssuanceFactory.sol

```diff
@@ -276,6 +292,7 @@ contract IssuanceFactory is CurrencyFactory{
 		return false;
   }
 
+	//TODO: why private? may be useful
```

Comment:
currencyMap itself is public so no need for this to be public

Reply:
sure, but this logic may be useful for validating issuance parameters before sending to network and waste gas if fail.